src/hockeypuck/conflux/recon/settings.go

```diff
@@ -49,7 +50,7 @@ type PTreeConfig struct {
 type Settings struct {
 	PTreeConfig
 
-	Version       string     `toml:"version"`
+	Version       string
```

Comment:
Will removing this annotation break compatibility with existing config files?

Reply:
No, the toml parser ignores unmapped keys. We haven't encountered any issues in testing.